modules/cortico/Login/Login.js

```diff
@@ -0,0 +1,169 @@
+import { Ellipsis } from "../../Icons/Ellipsis";
+import "./Login.css";
+import { CorticoIcon } from "../../Icons/CorticoIcon";
+import { create, getCorticoUrl } from "../../Utils/Utils";
+
```

Comment:
My concern with having the login box in the actual DOM long term is the Oscar application javascript can technically intercept it. We should probably be using a background worker for the authentication. But it's ok for now. The risk of Oscar being compromised is low, and would be very severe anyway.

Reply:
I agree, but I think for now this would be the fastest and easiest to do since I think we need this for diagnostic responses